common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,15 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Add Unsupported Problems and Tools"),
+        help=_(
+            "Enter true or false. If true, you can add unsupported content types to your course in Studio. "
+            "These content types are not recommended for use in courses due to non-compliance with one or "
+            "more of the base requirements, such as testing, accessibility, internationalization, and documentation."
```

Comment:
@cahrens @sstack22 Per a conversation we had yesterday we also need to update the help text under the Advanced Module policy to indicate that these types won't be available in your course unless you have specified a value of true  for "Add Unsupported Problems and Tools".